src/hashes/blake2b.rs

```diff
@@ -1,23 +1,21 @@
 // Copyright 2020 IOTA Stiftung
 // SPDX-License-Identifier: Apache-2.0
 
-use blake2::VarBlake2b;
+use blake2::{Blake2b, Digest};
```

Comment:
`Digest` is not used, except for `new()`, instead of `Blake2b::new()` we can use `Blake2b::default()`.